lib/utils.js

```diff
@@ -16,7 +16,7 @@ var CONFIG = {};
 function pointsFromESDocs(hits, timeField, idField) {
     return hits.map(function(hit) {
         var point = hit._source;
-        var time = new Date(point[timeField]).getTime();
+        var time = new Date(point[timeField]).toISOString();
```

Comment:
Given that `toNative` will handle a time that's a `Date`, how about we avoid the double conversion to and from strings and just leave the field as a Date?